manifest/main.js

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
Comment about the code naming/organization, not this specific line. I'd suggest using a different term than "manifest" to describe the code introduced here. We have an existing manifest concept that's unrelated, and the code here also deals in S3 inventory manifests, which have little to do with the index that's produced by the code. "Indexer" would be a good term that's descriptive of the concept and activity and unused in the codebase currently so not overloaded.

Reply:
Sounds good. I've used `./resrouceIndexer/*` for the code which creates the index, and `./src/resourceIndex.js` for the code which consumes it.